modules/puzzle/position_list.py

```diff
@@ -109,7 +111,7 @@ def is_complete(self, category, color, first_node, first_val):
                 if (self.material_difference() > 0.2
                         and abs(self.material_difference() - first_val) > 0.1
                         and first_val < 2
-                        and self.evaluation.mate is None
+                        and self.evaluation.mate() is None
```

Comment:
I see that in other place you used not is_mate() and here mate() is None, is there any difference? Probably best to always use the same to keep consistency. Not sure what do you think

Reply:
I intended for the code to look very similar to previous version to easily spot changes and do the refactoring in the next iteration. I would keep the code as it is for now. What do you think?

Reply:
Sounds good.